docs/user-guide/sdh/release-hub.md

```diff
@@ -0,0 +1,238 @@
+
+6. Before freezing the requirements, make sure the release order is correct, add applications if needed, and include environments in tenants (if not done already). Once you have finalized them, click **Lock Requirements**. 
+
+    ![Figure 15: Locking Requirements](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/freeze-requirements.gif)
+
+{% hint style="info" %}
+Once you lock the requirements, Devtron will prevent any unsolicited modifications to your release by anyone (including you). However, you can re-edit it by clicking **Unlock To Edit**.
+{% endhint %}
+
+---
+
+## Deploying Release
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to deploy a release.
+{% endhint %}
+
+{% hint style="info" %}
+### Prerequisite
+All your requirements need to be locked and [tenants](./tenants.md) must be configured.
+{% endhint %}
+
+This involves the deployment of the release to the specified tenant installations.
+
+1. Go to the **Rollout Release** tab.
+
+    ![Figure 16: Rollout Release Page](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/rollout-release.jpg)
+
+2. Your release needs to be marked as ready to proceed further. If it isn’t, you can mark it **Ready for release** from this screen.
+
+    ![Figure 17: Marking a Release as Ready](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/ready-release.gif)
+
+
+    Optionally, you can also do so by changing the status from **Draft** state to **Ready for release** within your release track. 
+
+    ![Figure 18: Alternative Way of Marking](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/ready-for-release.jpg)
+    
+3. Use the checkbox to select the applications belonging to the first release stage. You may use the filters on the left-hand side to make it easier.
+
+    {% embed url="https://www.youtube.com/watch?v=HPYZ4-hhoDM" caption="First Release Stage" %}
+
+4. Click **Deploy**.
+
+    If the application workflow has pre-deployment/post-deployment stage, you get a dropdown where you can specifically trigger either pre-deployment, deployment, or post-deployment stage.
+
+    ![Figure 19: Pre/Post Deployment](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/trigger-deployment.gif)
+
+5. Once the applications from the first release stage are successfully deployed, select the applications from the subsequent release stage and deploy.
+
+    {% embed url="https://www.youtube.com/watch?v=ioGZP1CCuxI" caption="Last Release Stage" %}
+
+{% hint style="warning" %}
+An application can be deployed on the tenant in the next release stage only if other applications in the previous stage are deployed successfully on the given tenant.
+{% endhint %}
+
+Here we covered the process of performing a production installation on one tenant. Similarly, you can per
+
+---
+
+## Extras
+
+### Viewing Rollout Status
```

Comment:
The rollout status has some grouping of statuses, when a tenant is in yet to trigger, when ongoing and failed. We need to cover that in doc